rotkehlchen/accounting/mixins/event.py

```diff
@@ -76,11 +76,11 @@ def process(
         ...
 
     @abstractmethod
-    def serialize(self) -> dict[str, Any]:
+    def serialize(self, to_debug: bool) -> dict[str, Any]:
```

Comment:
This `to_debug` flag is something that I am not sure about. But I added it because when exporting a json with events for debugging, `extra_data` is needed there. While in other places it is not.

Reply:
it is also needed in liquity transactions for the staking page